From bd081a1ae125c7c6b2cfec89746d1298a306ad78 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@debian.org>
Date: Tue, 16 Jan 2018 17:38:36 +0000
Subject: Fix putty-transfer regression test

The test key file is still called putty.rsa2, not putty.rsa.

Forwarded: no
Last-Update: 2018-01-16

Patch-Name: fix-regress-putty-transfer.patch
---
 regress/putty-transfer.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/regress/putty-transfer.sh b/regress/putty-transfer.sh
index 32c79f9e..57e46540 100644
--- a/regress/putty-transfer.sh
+++ b/regress/putty-transfer.sh
@@ -15,7 +15,7 @@ for c in 0 1 ; do
 	    ${OBJ}/.putty/sessions/compression_$c
 	echo "Compression=$c" >> ${OBJ}/.putty/sessions/kex_$k
 	env HOME=$PWD ${PLINK} -load compression_$c -batch \
-	    -i putty.rsa cat ${DATA} > ${COPY}
+	    -i putty.rsa2 cat ${DATA} > ${COPY}
 	if [ $? -ne 0 ]; then
 		fail "ssh cat $DATA failed"
 	fi
@@ -26,7 +26,7 @@ for c in 0 1 ; do
 		rm -f ${COPY}
 		dd if=$DATA obs=${s} 2> /dev/null | \
 			env HOME=$PWD ${PLINK} -load compression_$c \
-			    -batch -i putty.rsa \
+			    -batch -i putty.rsa2 \
 			    "cat > ${COPY}"
 		if [ $? -ne 0 ]; then
 			fail "ssh cat $DATA failed"
